apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -3261,8 +3262,14 @@ public static Bundle createBundle(
       List<IBaseResource> resourcesSubList = resources.subList(paging.getStartIndex(), endIndex);
       bundle = TransformerUtils.addResourcesToBundle(bundle, resourcesSubList);
       paging.setTotal(resources.size()).addLinks(bundle);
+
+      // Add number of paginated resources to MDC logs
+      LoggingUtils.logResourceCountToMdc(resourcesSubList.size());
```

Comment:
any reason not to be consistent here and use bundle.getTotal()?

Reply:
because the bundle's total is not set to the page size. So we have to take the resource sublist size instead.

Reply:
I never knew that! I was hoping we could derive it from the bundle object itself but I don't see anything else on that object that looks suitable so this looks like the way to go.